packages/kbn-custom-integrations/src/components/create/form.tsx

```diff
@@ -186,8 +186,7 @@ export const CreateCustomIntegrationForm = ({
               <EuiFlexItem grow={false}>
                 <EuiIconTip
                   content={i18n.translate('customIntegrationsPackage.create.dataset.name.tooltip', {
-                    defaultMessage:
-                      'Provide a dataset name to help organise these custom logs. This dataset will be associated with the integration. ',
+                    defaultMessage: 'The name of the dataset associated with this integration. ',
```

Comment:
Do we expect users to understand what a dataset is at this point? If not, it might be helpful to explain why they might want to name it a certain way.

Reply:
I think we should assume users do not understand it yet at this point.

Reply:
Yes, that might be good. How about we add something like "This name will be used in the name of the Elasticsearch data stream."?